### DISCUSSION TITLE
I have lost 2FA and forgot the password

### DISCUSSION THREAD

**Discussion by shubh5238** [community/community#66062] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have lost 2FA and forgot the password, please help me reseting the account.

**Comment by ghostinhershell** [comment]
Hey there @shubh5238, 

Thanks so much for reaching out in our community. If you lose access to your two-factor authentication credentials, you can use your recovery codes, or another recovery option, to regain access to your account. You can learn more here: [Recovering your account if you lose your 2FA credentials](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials).

Cheers!

**Comment by shubh5238** [comment]
I have lost recovery codes too. I am talking about a different account. I tried unlinking the email but for that need to login i guess. Please help.

**Reply by kuroodo** [reply]
Unfortunately you've lost your account for good.
Welcome to 2FA!

**Reply by shubh5238** [reply]
Is there a way i can unlink the email and create a new account?

**Reply by kuroodo** [reply]
Check if this helps: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/unlinking-your-email-address-from-a-locked-account

Otherwise you need to contact support and request they unlink your email

**Comment by shubh5238** [comment]
Is there a way i can unlink the email and create a new account?

**Reply by PhilipPartsch** [reply]
Is this helping you?
https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#unlinking-your-email-address
